crates/biome_module_graph/tests/snapshots/test_export_default_function_declaration.snap

```diff
@@ -18,16 +17,7 @@ export default function Component(): JSX.Element {}
 Exports {
   "default" => {
     ExportOwnExport => JsOwnExport(
-      sync Function "Component" {
-        accepts: {
-          params: []
-          type_args: []
-        }
-        returns: "JSX.Element" {
-          resolved: unknown
-          type_args: No types
-        }
-      }
+      Module TypeId(0)
```

Comment:
It seems that we "lost" some information, because they are now stored inside the typo info crate. However, it would be nice to have some tests inside `biome_js_type_info` to show that we don't have regressions around JsDoc.

Reply:
JSDoc comments aren't stored in the type information. They're still in these snapshots, you'll find one a few lines below.

I did wonder whether the module graph snapshots should also dump the registered types. Maybe it would be better indeed.